include/verilated_types.h

```diff
@@ -593,19 +593,28 @@ class VlQueue final {
         return v;
     }
 
-    // Setting. Verilog: assoc[index] = v
-    // Can't just overload operator[] or provide a "at" reference to set,
-    // because we need to be able to insert only when the value is set
-    T_Value& at(int32_t index) {
+    // Setting. Verilog: assoc[index] = v (should only be used by dynamic arrays)
+    T_Value& atWrite(int32_t index) {
         // cppcheck-suppress variableScope
         static thread_local T_Value t_throwAway;
         // Needs to work for dynamic arrays, so does not use T_MaxSize
-        if (VL_UNLIKELY(index < 0 || index >= m_deque.size())) {
+        if (VL_UNLIKELY(index < 0) || VL_UNLIKELY(index >= m_deque.size())) {
```

Comment:
```suggestion
        if (VL_UNLIKELY(index < 0 || index >= m_deque.size())) {
```

While GCC allows subexpression UNLIKELY, doing the whole expression is what C++20 will support with `[[likely]]`